attestation-service/src/main/kotlin/AttestationService.kt

```diff
@@ -439,7 +593,7 @@ object NoopAttestationService : AttestationService() {
  * @param iosAttestationConfiguration IOS AppAttest configuration.  See [IOSAttestationConfiguration] for details.
  * @param clock a clock to set the time of verification (used for certificate validity checks)
  * @param verificationTimeOffset allows for fine-grained clock drift compensation (this duration is added to the certificate
- * validity checks); can be negative.
+ * validity checks; can be negative.
```

Comment:
Nitpick: the removed `)` should be there. Also in -457 and -592.

Reply:
fixed